spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/AddFilesProcedure.java

```diff
@@ -60,7 +60,8 @@
       ProcedureParameter.required("table", DataTypes.StringType),
       ProcedureParameter.required("source_table", DataTypes.StringType),
       ProcedureParameter.optional("partition_filter", STRING_MAP),
-      ProcedureParameter.optional("check_duplicate_files", DataTypes.BooleanType)
+      ProcedureParameter.optional("check_duplicate_files", DataTypes.BooleanType),
+      ProcedureParameter.optional("list_partition_parallelism", DataTypes.IntegerType)
```

Comment:
on ExpireSnapshots and RemoveOrphans we have the parallelism parameter named

max_concurrent_xxxxxx

So maybe we just keep that convention? "max_concurrent_reads"?

Reply:
Should we be adding this to the Migrate Procedure as well?

Reply:
> on ExpireSnapshots and RemoveOrphans we have the parallelism parameter named
> 
> max_concurrent_xxxxxx
> 
> So maybe we just keep that convention? "max_concurrent_reads"?

ok , how about  `max_concurrent_list_partition`  or `max_concurrent_read_datafiles`

Reply:
> Should we be adding this to the Migrate Procedure as well?

Thanks for your advice, can you help where the  ` Migrate Procedure` code is?

Reply:
https://github.com/apache/iceberg/blob/7fc8f9a6c8fe9dacb2c19fdfa76dd8c592c47c24/spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/MigrateTableProcedure.java#L37

Reply:
I would consider "max_concurrent_read_datafiles" before list partition, since list partition doesn't really mean anything. I think reads may be fine but read_datafiles is probably also ok.

Reply:
Ok, Thanks